titapapunne/MoreThanWords
# #47: calculate progress bars and display screens for Associate and Partner Pod

- in the endpoint /calcProgressPodScreen in app.py, extracted current pod from request arguments so that could get data for specific pod screen user is on (not hardcoded to just TraineePod anymore)
- In PodScreen.js, put in argument when calling the endpoint for fetch (change /calcProgressPodScreen to /calcProgressPodScreen?pod=${pod})
- Deleted the other pod component screens (AssociatePod.js and PartnerPod.js) and renamed TraineePod.js to PodScreen.js since can just reuse all the code from that file for each pod. Updated these names in App.js stack navigator and deleted RandomScreen.js

Steps to test PR: 
- Login on the app using Yichen or Punpun's account to test having real user data
- Home screen should have trainee outcomes progress bar loaded.
- Then after clicking on any Pod from Home Screen, should be able to load all the progress bars with the correct number of total outcomes and completed outcomes.
- Then after clicking on any of the focus areas inside Trainee Pod, should be able to go to correct outcomes screen with correct tasks

Standard Checklist:
- [ x ] Does the code work? Does it perform its intended function, the logic is correct etc.
- [ x ] Have you merged the most recent version of staging into your branch?
- [ x ] Are all merge conflicts resolved? 
- [ x ] Can any logging or debugging code be removed? (Any debugging statements should be removed)
- [ x ] Is the `diff` the **minimum amount** needed to achieve this ticket's goal?
